### PR TITLE
Update docs for quality metrics

### PR DESCRIPTION
**Pull request by JaerongA**
- [x] Fix some doc errors in the docstring.
- [x] Added docstrings for quality metric tables. 
- [x] Updated `concepts.md`
- [x] Bump up a version.
- [x] Replace diagrams.

### CODE REVIEW

**Review by kabilar — CHANGES_REQUESTED** [review]
Thanks @JaerongA.  Please update to include rendering of the quality metrics Jupyter notebook in `workflow-array-ephys` on the docs page (e.g. see https://datajoint.com/docs/elements/element-array-ephys/0.2/tutorials/00-data-download-optional/).

File: requirements.txt [hunk 1]
```diff
@@ -5,4 +5,7 @@ openpyxl
 plotly
 pyopenephys>=1.1.6
 seaborn
-scikit-image
\ No newline at end of file
+scikit-image
+spikeinterface
+spikeextractors
```
Comment (kabilar) [inline requirements.txt:10]:
```suggestion
```

Comment (kabilar) [reply requirements.txt:10]: I could be mistaken, but I don't believe that `spikeextractors` is required for this package.

File: docs/src/concepts.md [hunk 2]
```diff
@@ -156,6 +156,9 @@ Tables for storing probe or unit-level visualization results.
 | --- | --- |
 | ProbeLevelReport | A table to store drift map figures generated from each recording probe. |
 | UnitLevelReport | A table to store figures (waveforms, autocorrelogram, peak waveform + neighbors) generated for each unit. |
+| QualityMetricCutoffs | A table to store cut-off values for cluster quality metrics. |
+| QualityMetricSet | A table to store a set of quality metric values for clusters and its cut-offs. |
```
Comment (kabilar) [inline docs/src/concepts.md:160]:
```suggestion
| QualityMetricSet | A manual table to match a set of quality metric values for clusters with desired cut-offs. |
```

File: element_array_ephys/ephys_report.py [hunk 3]
```diff
@@ -226,6 +243,13 @@ class QualityMetricSet(dj.Manual):
 
 @schema
 class QualityMetricReport(dj.Computed):
+    """Table for storing quality metric figures.
+
+    Attributes:
+        QualityMetricSet (foreign key): QualityMetricSet primary key.
+        plot_grid (longblob): Figure object.
```
Comment (kabilar) [inline element_array_ephys/ephys_report.py:250]:
```suggestion
        plot_grid (longblob): Plotly figure object.
```

**Review by JaerongA — COMMENTED** [review]
Thanks @kabilar for the feedback. Your suggestions have been committed. I'll add a new figure soon.

File: images/attached_array_ephys_element_acute.svg
Comment (kabilar) [inline images/attached_array_ephys_element_acute.svg:1]: Please add `SkullReference` to this diagram.

File: images/attached_array_ephys_element_chronic.svg
Comment (kabilar) [inline images/attached_array_ephys_element_chronic.svg:1]: Please add `SkullReference` to this diagram.

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,12 @@
 
 Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and
  [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) convention.
+## [0.2.5] - 2023-04-12
+
++ Add docstrings for quality metric tables.
++ Fix the docstring errors.
++ Update `concepts.md`
++ Update schema diagrams.
```
Comment (kabilar) [inline CHANGELOG.md:10]:
```suggestion

## [0.2.5] - 2023-04-12

+ Add - docstrings for quality metric tables
+ Fix - docstring errors
+ Update - `concepts.md`
+ Update - schema diagrams with quality metrics tables
```

**Comment by kabilar** [comment]
Thanks for updating the diagrams.  Please add the `SkullRerefence` table to all diagrams.

File: CHANGELOG.md
Comment (kabilar) [inline CHANGELOG.md:1]: Please add the following to the end of this file:
```
[0.2.5]: https://github.com/datajoint/element-array-ephys/releases/tag/0.2.5
```

**Comment by kabilar** [comment]
Thanks @JaerongA. Unfortunately, the table name is showing up as it is defined on the database. See image below.

image

Ideally it should be displayed as `lab.SkullReference`.  Try running `from workflow_array_ephys.pipeline import *` and then displaying the diagram.

**Comment by JaerongA** [comment]
> Thanks @JaerongA. Unfortunately, the table name is showing up as it is defined on the database. See image below.
> 
> image
> 
> Ideally it should be displayed as `lab.SkullReference`. Try running `from workflow_array_ephys.pipeline import *` and then displaying the diagram.

Yeah, I've tried this and that.. and it's still showing it.

**Comment by JaerongA** [comment]
@kabilar I've updated the diagram, but it's showing as `SkullReference`, not `lab.SkullReference`.

**Pull request merged**